test/wappsto_test.py

```diff
@@ -85,6 +85,17 @@ def create_response(self, verb, callback_exists, trace_id):
     return '{"jsonrpc": "2.0", "id": "1", "params": {"url": "'+url+'",'+trace+' "data": {"meta": {"id": "'+id+'"}, "data": "93"}}, "method": "'+verb+'"}'
 
 
+def get_expected_json(self):
```

Comment:
What are you doing in this function?

Reply:
it gets the JSON file and reads its information, after that it modifies data so it stores the same information as is being expected to be sent